containers/init/clone/clone.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
I think it's safer to always use "set -ex" to make sure that failures from the shell script are always propagated.
As this script gets updated, it's going to be easy to miss the fact that newly added failed command can actually fail silently.

Reply:
You're right, much shorter script too... thanks.